.github/workflows/extension_ci.yml

```
@@ -151,25 +152,27 @@ jobs:
           trunk publish pgmq --version ${pgmq_ver} --description "A lightweight message queue. Like AWS SQS and RSMQ, on Postgres." --homepage "https://github.com/tembo-io/pgmq" --repository "https://github.com/tembo-io/pgmq" --license "PostgreSQL" --category featured --category orchestration
   build_and_push:
     name: Build and push images
-    # needs:
-    #   - publish
     runs-on:
       - self-hosted
       - dind
       - large-8x8
+    strategy:
+      matrix:
```

```suggestion
      fail-fast: false
      matrix:
```
I'm interested in how it behaves with other versions